### ISSUE TITLE
Argument 'label_block' => false not working for Media Controls since version 2.9

### ISSUE THREAD

**Issue by black-eye** [elementor/elementor#11756]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

After updating from version 2.8, I've realized that the parameter `'label_block' => false` doesn't work for Media controls anymore. Instead of the small image, there's a blank space there:

![obrazek](https://user-images.githubusercontent.com/5634870/86013899-f4adba00-ba1f-11ea-9a41-43625db487da.png)

Here is the code I use:

```
$this->add_control(
	'testimonial_image',
	[
		'label' => 'Profile Image',
		'type' => \Elementor\Controls_Manager::MEDIA,
		'label_block' => false,
	]
);
```

When I delete the label_block argument, full-width image thumb is back, but to save the space I'd like to have there a smaller one.

## Steps to reproduce

Create a widget with the control as showed above and test it in the editor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by black-eye** [comment]
The issue is still there in 3.0.x

@shilo-ey, @arielk, could you pls bring some light to the change of this argument? Has it been removed on purpose?
The argument is still mentioned in the docs - https://developers.elementor.com/elementor-controls/media-control/

If it was removed, it would be a pity, it was a nice feature, I used it quite often in my widgets, when I needed to add smaller images like avatars and icons and wanted to save some space.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.12

Feel free to update

Thanks!
